MantidQt/CustomInterfaces/inc/MantidQtCustomInterfaces/Reflectometry/ReflSettingsTabPresenter.h

```diff
@@ -72,6 +73,8 @@ class MANTIDQT_CUSTOMINTERFACES_DLL ReflSettingsTabPresenter
   IReflSettingsTabView *m_view;
   /// The main presenter
   IReflMainWindowPresenter *m_mainPresenter;
+  /// Name of the current instrument in use
+  std::string currentInstrumentName;
```

Comment:
If this is a member variable it should be renamed to `m_currentInstrumentName`. See [here](http://www.mantidproject.org/C%2B%2B_Coding_Standards#Naming_Conventions) for more naming conventions in the project.